### PR TITLE
add node to title types

### PR DESCRIPTION
**Pull request by brumeregan**
currently title for <Selector /> component needs to be only string, but it also can render node
so to prevent type errors in console, it is better to add this type also :)

### CODE REVIEW

File: src/Selector/Selector.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Selector extends WixComponent {
       'cinema',
     ]),
     imageShape: PropTypes.oneOf(['rectangular', 'circle']),
-    title: PropTypes.string.isRequired,
+    title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).isRequired,
```
Comment (argshook) [inline src/Selector/Selector.js]:
```suggestion
    title: PropTypes.node.isRequired,
```

PropType.node is also PropType.string, so there's no need of `oneOfType`

Comment (brumeregan) [reply src/Selector/Selector.js]: hmm.. but I have error when run my tests..

Comment (argshook) [reply src/Selector/Selector.js]: what kind of error? Prop incompatibility should be a warning

Comment (argshook) [reply src/Selector/Selector.js]: here's the implementation of `PropTypes.node` checker: https://github.com/facebook/prop-types/blob/master/factoryWithTypeCheckers.js#L474-L479

as you see `string` returns `true`. If you get a warning i would assume it's from somewhere else

Comment (brumeregan) [reply src/Selector/Selector.js]: I see this:

> console.error node_modules/prop-types/checkPropTypes.js:20
      Warning: Failed prop type: Invalid prop `title` of type `object` supplied to `Selector`, expected `string`.

Comment (argshook) [reply src/Selector/Selector.js]: okay, so this happens before this PR.

with `PropTypes.node.isRequired` you will not have such warning

**Pull request merged**